CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Change Log
 
+## To be released
+
+### Repository Changes
+
+...
+
+### New Features
+
+#### honeycomb-core
+
+- `Orbit<'a, const N_MARKS: usize, T: CoordsFloat>` - Generic implementation for
+  2D orbit computations. The structure itself only contains meta-data, the orbit
+  computation is done through the `Iterator` trait implementation.
+- `OrbitPolicy<'a>` - Enum used to specify the beta functions used by an orbit.
+  It currently does not not support compositions.
+- New (temporary?) method for `TwoMap`: `beta_bis`. It works exactly like the
```

Comment:
`beta_bis` name is not very good.
I would rather have `beta_rt` or `beta_runtime` and `beta_ct` (compile time?) to have an clearer idea of whats going on. Or `beta_generic` or `beta_dispatch`.

Reply:
I renamed `beta_bis` to `beta_runtime` and cahnged its implementation to use a match block. I don't wan't to rename the original beta function at the moment though.